### DISCUSSION TITLE
I have runned: `pip install pytorch-lightning`, but get error: No module named `pytorch_lightning`

### DISCUSSION THREAD

**Discussion by muyuuuu** [Lightning-AI/pytorch-lightning#11498] [General]
![image](https://user-images.githubusercontent.com/43681138/149628558-289611ed-244e-4ed7-95d0-c34d4e341520.png)

python: 3.8.8
pytorch: 1.7.0
pytorch-lightning: 1.5.8
windows 10 and anaconda 4.11.0

**Comment by akihironitta** [comment]
~Duplicate of #11498.~
Duplicate of #11499.

**Comment by rohitgr7** [comment]
maybe your pip is not referencing to the python installed on your system.
Either try using an env like conda or virtualenv
or install lightning using
```py
python -m pip install pytorch-lightning
python train.py
```

**Reply by muyuuuu** [reply]
Some error: D:\users\anaconda3\python.exe: No module named pip.__main__; pip is a package and cannot be directly executed.

**Reply by rohitgr7** [reply]
yes, that means pip isn't installed for your python which you are using to execute the script and `pip` installation you used to install must be referring to something else.

can you post?
```py
which pip
which python
```

also try to reinstall python or better I'd recommend using envs.

**Reply by muyuuuu** [reply]
emmm, I use conda to create a new virtual env, and pip install in env, this problem has been solved. thank you.

**Reply by saju8652** [reply]
@muyuuuu can you tell me how your problem got solved?

**Comment by muyuuuu** [comment]
here is pip list

![image](https://user-images.githubusercontent.com/43681138/149702881-724c5a7c-641c-4eb4-9582-b0e048a4b263.png)
